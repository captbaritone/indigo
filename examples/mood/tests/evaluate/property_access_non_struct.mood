pub fn test(): i32 {
    let b: i32 = 10_i32;
    b.prop
}