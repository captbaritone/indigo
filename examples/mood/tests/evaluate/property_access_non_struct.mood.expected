-----------------
INPUT
----------------- 
pub fn test(): i32 {
    let b: i32 = 10_i32;
    b.prop
}
-----------------
OUTPUT
-----------------
Error: Cannot access member of non-struct type.:
 --> property_access_non_struct.mood:3:5

  |     let b: i32 = 10_i32;
3 |     b.prop
  |     ^ Type is i32
  | }
